### ISSUE TITLE
关于代码中的一些问题

### ISSUE THREAD

**Issue by mayidu** [huanyuwu/video-condensation#1]
我看你说这个是你的硕士毕业论文？你的文章与video condensation by ribbon carving文章的不同之处在哪儿呢？我看了一下给出的效果相似，你们这个是否属于视频浓缩的范畴呢？

I guess you are Chinese, are you? your code can't run, can you tell me which version of matlab you used?Thanks!

**Comment by huanyuwu** [comment]
Hi,
Do you have error messages you can share? If you can compile the C code and generate its MEX file, it should run with proper avi files. The MATLAB version I used was a bit old, so it's possible that you may want to replace some older functions (likely video reading) if it's not working in your version.

**Comment by mayidu** [comment]
Yes! When I use MATLAB 2017, there are many functions not work. And I modified the functions, but there are too many! Now, I can get flex0_cost.avi and flex0_video.avi, and others video I don't get!
Is video synopsis of your work? Can you give me some test videos? Thanks

**Comment by huanyuwu** [comment]
Have you tried whole_cost.avi and whole_video.avi files? They can be downloaded using GIT LFS.
I suppose the errors you have are mainly related to video reading/writing. If you can find corresponding functions in newer version of MATLAB it should run okay.

**Comment by mayidu** [comment]
Can you send a video which you get to my Email which is mayidu@126.com? thaks!
